### DISCUSSION TITLE
Couldn't/cant start Wi-Fi with Bookworm image build...

### DISCUSSION THREAD

**Discussion by shaunhurley** [guysoft/FullPageOS#558] [Q&A]
So this is a weird one...I have quite a number of devices with FullpageOS and have been working it for going on five years now (I literally have four of them running on my test bench as I type) so I'm pretty familiar with the mechanics at this point.

Over the last couple of days I've been playing with the latest Bookworm build to work around an unrelated issue and I discovered that I could not get the image/device to connect to my Wi-Fi network by configuring the network through the fullpageos-wpa-supplicant.txt file.  I've had the device connected view ethernet cable to troubleshoot.

```
## WPA/WPA2 secured
network={
  ssid="<theSSID>"
  psk="<thepassword>"
}
```

I initially thought it might be hardware related - I was testing on a new Pi 5, but I had the same behavior running the image on a Pi 4 and a Pi 3B+.  When using the standard Raspian OS desktop Bookworm image an a different SD card the network Wi-Fi worked fine.

I double checked the edits (no typos), copied the supplicant file from another, working device, and even pulled down a fresh image and re-flashed a spare SD-Card and did nothing else except edit the supplicant settings, to no avail.

Following some random googling, I came across the Network Manager as an alternative and was able to get the Wi-Fi to come up using the Network Manager CLI with the following commands:

```
sudo nmcli radio wifi on
sudo nmcli dev wifi connect "<theSSID>" password "<thepassword"
```

Note: the radio was actually already enabled.

Anyway, device working, and the network config via this mechanism is 'sticky' so it persists through a reboot, but I'm just confused as to why the supplicant config isn't working.  Happy to explore any thoughts from the hive mind!

Cheers,

Shaun

**Comment by guysoft** [comment]
There is an issue:
https://github.com/guysoft/CustomPiOS/issues/219

Its caused because Rpi broke support of the feature by changing dhcp client without alerting:
https://github.com/raspberrypi/bookworm-feedback/issues/72
